common/buildcraft/robotics/ZoneChunk.java

```diff
@@ -4,24 +4,29 @@
  * of the license located in http://www.mod-buildcraft.com/MMPL-1.0.txt */
 package buildcraft.robotics;
 
-import java.util.BitSet;
-import java.util.Random;
+import buildcraft.api.core.INetworkLoadable_BC8;
+import buildcraft.core.lib.utils.BitSetUtils;
+import buildcraft.core.lib.utils.NetworkUtils;
 import io.netty.buffer.ByteBuf;
-
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.util.math.BlockPos;
 
-import buildcraft.api.core.ISerializable;
-import buildcraft.core.lib.utils.BitSetUtils;
-import buildcraft.core.lib.utils.NetworkUtils;
+import java.util.BitSet;
+import java.util.Random;
 
-public class ZoneChunk implements ISerializable {
+public class ZoneChunk implements INetworkLoadable_BC8 {
```

Comment:
Your misusing ````INetworkLoadable_BC8```` slightly - can you add the generic type so we get the proper type returned?